### PR TITLE
HEEDLS-631 Add migration to reduce the GroupDescription column length

### PR DESCRIPTION
**Pull request by AlexJacksonDS**
### JIRA link
https://softwiretech.atlassian.net/browse/HEEDLS-631

### Description
Migration to reduce the size of Groups.GroupDescription from nvarchar(max) to nvarchar(1000), truncating any values over this limit (there should hopefully still be none by the time this makes it live)

### Screenshots
N/A

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [x] Run the formatter and made sure there are no IDE errors.
- [ ] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [x] Manually tested my work with and without JavaScript. Full manual testing guidelines can be found here: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/6703648740/Testing
- [ ] Updated/added documentation in Swiki and/or Readme. Links (if any) are below:
  - [doc_1_here](link_1_here)
- [ ] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken.
- [x] Scanned over my own MR to ensure everything is as expected.

### CODE REVIEW

**Review by JBsoftwire — APPROVED** [review]
Looks good to me.

File: DigitalLearningSolutions.Data.Migrations/202111081225_ReduceGroupsGroupDescriptionLength.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+{
+    using FluentMigrator;
+
+    [Migration(202111081225)]
+    public class ReduceGroupsGroupDescriptionLength : Migration
+    {
+        public override void Up()
+        {
+            Execute.Sql("UPDATE Groups SET GroupDescription = LEFT(GroupDescription, 1000)");
+            Alter.Table("Groups").AlterColumn("GroupDescription").AsString(1000).Nullable();
```
Comment (SteveJacksonSoft) [inline DigitalLearningSolutions.Data.Migrations/202111081225_ReduceGroupsGroupDescriptionLength.cs:11]: Does this specify the number of characters or the number of bytes?

Comment (AlexJacksonDS) [reply DigitalLearningSolutions.Data.Migrations/202111081225_ReduceGroupsGroupDescriptionLength.cs:11]: it'll set it to a nvarchar(1000)

Comment (SteveJacksonSoft) [reply DigitalLearningSolutions.Data.Migrations/202111081225_ReduceGroupsGroupDescriptionLength.cs:11]: Grand. Are you confident you can fit a 1000-character string in there? I'm slightly hazy on this, especially where character encodings are concerned, but if you're confident / have tested then merge away.

Comment (AlexJacksonDS) [reply DigitalLearningSolutions.Data.Migrations/202111081225_ReduceGroupsGroupDescriptionLength.cs:11]: Yes, I've put a 1000 character string xxxxxxxxx... into mine, and it truncated the longer one down.

**Pull request merged**